### PR TITLE
Create nf-syn53689508-s3.yaml

### PR DESCRIPTION
**Pull request by jaybee84**
This PR aims to provision an S3 bucket to copy files from GEO to Synapse.

### CODE REVIEW

File: config/prod/nf-syn53689508-s3.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.0/templates/S3/synapse-external-bucket.j2"
+stack_name: "nf-syn53689508-s3"
+stack_tags:
+  OwnerEmail: 'jineta.banerjee@sagebionetworks.org'
+  CostCenter: "NTAP NF Addendum 4 / 301100"
```
Comment (jesusaurus) [inline config/prod/nf-syn53689508-s3.yaml:8]: It looks like this isn't an [active cost center](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2553544733/Finance+Program+Codes) anymore, but `NTAP NF Amendment 5 / 301101` and `NTAP NF Amendment 6 / 301102` both are. Are either of those two cost centers appropriate here?

**Review by jesusaurus — CHANGES_REQUESTED** [review]

**Review by zaro0508 — APPROVED** [review]

File: config/prod/nf-syn53689508-s3.yaml (hunk 1 above)
Comment (jaybee84) [reply config/prod/nf-syn53689508-s3.yaml:8]: @jesusaurus I found this one in the program codes: `JHU NTAP / 301100`. Can we use that one?

Comment (jesusaurus) [reply config/prod/nf-syn53689508-s3.yaml:8]: @jaybee84 yeah, that works too

Comment (jaybee84) [inline config/prod/nf-syn53689508-s3.yaml:8]:
```suggestion
  CostCenter: "JHU NTAP / 301100"
```

Comment (jaybee84) [reply config/prod/nf-syn53689508-s3.yaml]: done

**Review by jesusaurus — APPROVED** [review]

**Pull request merged**